bebop_driver/CMakeLists.txt

```diff
@@ -131,26 +131,41 @@ add_dependencies(bebop_driver_node ${PROJECT_NAME}_gencfg)
 #   DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
 # )
 
-## Mark executables and/or libraries for installation
-# install(TARGETS bebop_autonomy bebop_autonomy_node
-#   ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-#   LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-#   RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
-# )
+# Mark executables and/or libraries for installation
+install(TARGETS ${PROJECT_NAME}_nodelet ${PROJECT_NAME}_node libbebop
+  ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+  RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+)
 
-## Mark cpp header files for installation
-# install(DIRECTORY include/${PROJECT_NAME}/
-#   DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
-#   FILES_MATCHING PATTERN "*.h"
-#   PATTERN ".svn" EXCLUDE
-# )
+# Copy ARSDK libraries
+install(DIRECTORY ${ARDRONESDK3_PATH}/lib/
```

Comment:
Installing Parrot SDK's libs into `${CATKIN_PACKAGE_LIB_DESTINATION}` might a bit problematic, since it also includes libraries like `libcurl.so` and `libjson.so`. I have been trying to package the SDK independently with the ultimate goal of removing the inline build step (#2). If you look at [the install rules there](https://github.com/AutonomyLab/parrot_arsdk/blob/indigo-devel/CMakeLists.txt#L67) you see that I copy all the products into a sub-folder. In your opinion, is it a safer approach?